packages/core/src/components/application/application.scss

```
@@ -19,6 +20,38 @@
+    @include fonts.typography-body;
+
+    position: absolute;
+    inset-block-start: -100%;
+    inset-inline-start: calc(
+      var(--ix-safe-area-inset-left, 0rem) +
+        var(--theme-focus--border-thickness) +
+        var(--theme-btn--focus--outline-offset)
+    );
+    z-index: calc(var(--theme-z-index-fixed) + 1);
+
+    padding: var(--theme-space-1) var(--theme-space-3);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)